test/library/packages/Itertools/check_Serial.good

```diff
@@ -0,0 +1,14 @@
+2.3 -3.4 8.91 12.04 2.8
+2.3 -3.4 8.91 12.04 2.8
+2.3 -3.4 8.91 12.04 2.8
+
+This is a test sentence.
+This is a test sentence.
+This is a test sentence.
+This is a test sentence.
+This is a test sentence.
+
+-3..21 by 3
+-3..21 by 3
+-3..21 by 3
+-3..21 by 3
```

Comment:
Before the latest commit, we had an extra newline, which would cause a failure in testing. This showed up in the diff as:

```
<
```

When running these tests locally now, I get failures with a diff of:

```
< -3..21 by 3
\ No newline at end of file
---
> -3..21 by 3
```

Are you able to put a newline character at the end of your good files, such that testing passes? See this [SO post](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) for more info.

Reply:
I have made the necessary changes, the tests should now pass.

It was a bit careless from my side. I wrote good files in vim earlier, but then switched to VS Code. The default newline character in VS was set to `\r\n`, which was creating the problem IMO.

Sorry for the trouble. Could you take a look now? Thanks.

Reply:
Sure thing. Will take a look today.